base_classes/NXdata.nxdl.xml

```diff
@@ -83,25 +83,28 @@
 		<doc>
 			.. index:: plotting
 			
-			String array that defines the independent data fields used in 
-			the default plot for all of the dimensions of the *signal* field
-			(the *signal* field is the field in this group that is named by
-			the ``signal`` attribute of this group). 
-			One entry is provided for every dimension in the *signal* field.
+			Array of strings holding the :ref:`names &lt;validItemName>` of
+			the independent data fields used in the default plot for all of
+			the dimensions of the :ref:`signal &lt;/NXdata@signal-attribute&gt;`
+			or :ref:`auxiliary signals &lt;/NXdata@auxiliary_signals-attribute&gt;`.
```

Comment:
Maybe, "as well as any" instead of "or".

Reply:
done